scripts/e2e_test/start_test.sh

```diff
@@ -274,17 +274,18 @@ install_necessary_tools() {
   if ! command -v unzip &> /dev/null
   then
     echo "installing unzip"
-    apt update && apt install -y unzip
+    apt update
```

Comment:
if the dockerfile has already install unzip, is there necessary to judge?

Reply:
> if the dockerfile has already install unzip, is there necessary to judge?

Yes, method `install_necessary_tools` could be removed now